package.json

```diff
@@ -7,7 +7,6 @@
     "local-docs": "npm-publish-scripts serve",
     "publish-docs": "npm-publish-scripts publish-docs",
     "publish-release": "npm run lerna-bootstrap && gulp test && lerna publish --concurrency 1",
-    "postinstall": "npm run lerna-bootstrap",
```

Comment:
Gotcha. AppVeyor appears to be done and all fine here. Waiting for Travis to complete.